.github/workflows/blackduck_scan.yml

```diff
@@ -42,10 +42,8 @@ jobs:
           --detect.code.location.name="${{ vars.BLACKDUCK_APP_ID }}-${{ vars.BLACKDUCK_PROJECT_ID }}-${VERSION_ID}" \
           --detect.source.path="." \
           --detect.clone.project.version.latest=true \
-          --detect.excluded.directories=appsec \
```

Comment:
Why we need to remove this 3 lines ?

Reply:
these options have been removed. https://sig-product-docs.synopsys.com/bundle/integrations-detect/page/properties/all-properties.html

Reply:
Ok!